help/tenacity.metainfo.xml.in

```diff
@@ -24,15 +24,15 @@
     <category>Audio</category>
   </categories>
   <content_rating type="oars-1.1" />
-  <url type="homepage">https://saucedacity.github.io/</url>
-  <url type="bugtracker">https://github.com/saucedacity/saucedacity/issues</url>
+  <url type="homepage">https://tenacityaudio.org/</url>
+  <url type="bugtracker">https://github.com/tenacityteam/tenacity/issues</url>
   <url type="help">https://matrix.to/#/#saucedacity:matrix.org</url>
-  <!-- <url type="translate">https://www.audacityteam.org/community/translators/</url> -->
+  <url type="translate">https://hosted.weblate.org/projects/tenacity/tenacity</url>
   <screenshots>
     <screenshot type="default">
-      <image type="source">https://saucedacity.github.io/saucedacity.png</image>
-    </screenshot>
-  </screenshots>
+        <image type="source">https://cdn.statically.io/gh/tenacityteam/assets/e134a1569714bde2973556d5ce273ab627e2ec5b/screenshots/tenacity-screenshot-light-3840x2160.png</image>
+        <image type="source">https://cdn.statically.io/gh/tenacityteam/assets/e134a1569714bde2973556d5ce273ab627e2ec5b/screenshots/tenacity-screenshot-dark-3840x2160.png</image>
```

Comment:
This may be affected during a transition to another platform.

Reply:
Our assets repo was moved to Codeberg, so I can probably fix that up with little to no issue.